src/AppBundle/Entity/User.php

```diff
@@ -13,7 +13,7 @@
  * @ORM\Table()
  * @ORM\Entity(repositoryClass="AppBundle\Entity\UserRepository")
  */
-class User implements UserInterface
+class User implements UserInterface, \JsonSerializable
```

Comment:
now return getFullName = return $this->getLastName() . ' ' . $this->getFirstName() . ' ' . $this->getMiddleName(); I don’t know what exactly return needed